core/src/test/java/io/grpc/internal/AbstractManagedChannelImplBuilderTest.java

```diff
@@ -61,15 +67,56 @@
         }
       };
 
-  private static final StatsContextFactory DUMMY_STATS_FACTORY =
-      new StatsContextFactory() {
+  private static final Tagger DUMMY_TAGGER =
```

Comment:
It probably won't hurt if this test uses the fake implementations from StatsTestUtils. Then you can save a lot of lines of the dummy implementations.